.github/workflows/ci.yml

```
@@ -12,7 +12,12 @@ jobs:
       fail-fast: false
       matrix:
         ruby-version: ['3.0', '3.1', '3.2', '3.3']
-        gemfile: ['rails_6.0','rails_6.1','rails_7.0', 'rails_7.1']
+        gemfile: ['rails_6.0','rails_6.1','rails_7.0', 'rails_7.1', 'rails_8.0']
```

Should this also include Rails 7.2?

```suggestion
        gemfile: ['rails_6.0','rails_6.1','rails_7.0', 'rails_7.1', 'rails_7.2', 'rails_8.0']
```

---

true, it should!